lambdas/adobe-sign-api/serverless.yml

```diff
@@ -28,6 +33,34 @@ functions:
             parameters:
               paths:
                 id: true
+  getTemplate:
+    role: lambdaRole
+    handler: lib/templates-controller.getTemplate
+    vpc:
+      securityGroupIds:
+        - { 'Fn::ImportValue': 'AdobeSign-ServerlessSecurityGroup-GroupId' }
+      subnetIds:
+        - { 'Fn::ImportValue': 'AdobeSign-PrivateSubnetA-Ref' }
+    events:
+      - http:
+          path: templates/{id}
+          method: get
+          request:
+            parameters:
+              paths:
+                id: true
+  createTemplate:
+    role: lambdaRole
+    handler: lib/templates-controller.createTemplate
+    vpc:
+      securityGroupIds:
+        - { 'Fn::ImportValue': 'AdobeSign-ServerlessSecurityGroup-GroupId' }
+      subnetIds:
+        - { 'Fn::ImportValue': 'AdobeSign-PrivateSubnetA-Ref' }
+    events:
+      - http:
+          path: templates/{id}
```

Comment:
Since this is a POST, we should get info from the request body instead of path params